src/west/manifest.py

```diff
@@ -152,7 +152,10 @@ def _mpath(cp: Optional[configparser.ConfigParser] = None,
     cfg.read_config(configfile=cfg.ConfigFile.LOCAL, config=cp, topdir=topdir)
 
     try:
-        return cp.get('manifest', 'path')
+        path = cp.get('manifest', 'path')
+        filename = cp.get('manifest', 'file', fallback=_WEST_YML)
```

Comment:
How will this behave if manifest project uses `myname.yml` but we import another project manifest that uses `west.yml` or `othername.yml` for their manifest ?

Reply:
the `import` key already supports specifying a relative path:
https://docs.zephyrproject.org/latest/guides/west/manifest.html#west-manifest-import-path

Reply:
@tejlmand please revisit

Reply:
thanks for the clarification.